{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -44,24 +47,27 @@ repos:
             'flake8-string-format',
             'flake8-tidy-imports',
         ]
-        args: ['--exclude=docs/conf.py', '--ignore=E203,E266,E501,W503', '--max-line-length=88', '--max-complexity=18', '--select=B,C,E,F,W,T4']
+        args: ['--ignore=E203,E266,E501,W503', '--max-line-length=88', '--max-complexity=18', '--select=B,C,E,F,W,T4']
 
 - repo: https://github.com/timothycrosley/isort
   rev: 5.10.1
   hooks:
       - id: isort
-        args: ['--multi-line=3', '--trailing-comma', '--force-grid-wrap=0', '--use-parentheses', '--line-length=88', '--skip=docs/conf.py']
+        exclude: docs/|setup.py|tests/
```

Comment:
what's the reason for adding setup.py and tests? don't see why we wouldn't want sorted imports on these files